arrow-libs/core/arrow-meta/build.gradle.kts

```diff
@@ -4,6 +4,10 @@ plugins {
   id("org.jetbrains.kotlin.kapt")
 }
 
+publishJVM {
+  isDokkaEnabled = false
```

Comment:
👌 So by default it sets up dokka, but you disable it it only uses Dokka for the MPP doc jars?